imagej/src/test/java/science/aist/imaging/service/imagej/imageprocessing/converter/ColorConverterTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 the original author or authors.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package science.aist.imaging.service.imagej.imageprocessing.converter;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import science.aist.imaging.api.domain.color.RGBColor;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * <p>Test class for {@link ColorConverter}</p>
+ *
+ * @author Christoph Praschl
+ * @since 1.1
+ */
+public class ColorConverterTest {
+
+    @Test
+    public void testConvert() {
+        // given
+        ColorConverter colorConverter = new ColorConverter();
+
+        // when
+        RGBColor convert = colorConverter.convert(1);
+
+        // then
+        Assert.assertNotNull(convert);
+        assertEquals(convert.getRed(), 0.0);
+        assertEquals(convert.getGreen(), 0.0);
+        assertEquals(convert.getBlue(), 1.0);
```

Comment:
I know it is a very small detail, but I kinda dislike, that one time the method is qualified with the class name, and the other three times not.

Reply:
Fixed in [739c19f](https://github.com/FHOOEAIST/imaging/pull/14/commits/739c19fa2d18dad40ebac4676254c36ff07279f0)